man/tpm2_commit.1.md

```diff
@@ -0,0 +1,82 @@
+% tpm2_commit(1) tpm2-tools | General Commands Manual
+
+# NAME
+
+**tpm2_commit**(1) - Performs the first part of an ECC anonymous signing
+operation.
+
+# SYNOPSIS
+
+**tpm2_commit** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_commit**(1) - Performs the first part of an ECC anonymous signing
+operation. The TPM will perform the point multiplications on the provided points
+and return intermediate signing values. The signHandle parameter shall refer to
+an ECC key and the signing scheme must be anonymous (TPM_RC_SCHEME).
+
+# OPTIONS
+
+  * **ARGUMENT**=_FILE_:
+
+    Specify the input data used to derive the x coordinate of the basepoint.
+
+  * **\--basepoint-y**=_FILE_:
+
+    Specify the y coordinate of the basepoint.
+
+  * **\--eccpoint-P**=_FILE_:
+
+    Specify a point on the curve used by sign handle.
```

Comment:
It looks like ARGUMENT, y and P can be omitted, what happens when they are omitted?

Reply:
These options give additional controls in determining the point on the curve. Defaults work just fine and that is why they are optional parameters. Should we go with the defaults for now?

Reply:
Retaining these options for now.